resources/migrations/000_migrations.yaml

```diff
@@ -7654,6 +7654,21 @@ databaseChangeLog:
             constraintName: fk_view_log_ref_user_id
             onDelete: CASCADE
 
+  - changeSet:
+       id: 267
+       author: sb
+       comment: Added 0.37.0
+       changes:
+         -  addColumn:
+              tableName: metabase_database
+              columns:
+                - column:
+                    name: start_of_week
+                    type: varchar(32)
+                    defaultValue: sunday
+                    constraints:
+                      nullable: false
```

Comment:
I'm confused why `start_of_week` is stored for each db. Do we set it during sync? It looks like each driver hardcodes a value.

Reply:
This is in preparation for when we do this per-db, but I've punted on this for v1 to limit scope.

Reply:
Should we remove the migration now and add it back when it's used? There always some risk that future features get punted indefinitely and then we have an extra column sitting around.

Reply:
if we're not using this yet, might as well remove the migration and pull the next changeset ids up. if you do this make sure to update #migrations on slack